test/cryptopunk/crypto/tron_test.exs

```diff
@@ -54,4 +54,33 @@ defmodule Cryptopunk.Crypto.TronTest do
       assert expected_address == Tron.address(key)
     end)
   end
+
+  test "test address validation (valid addresses)" do
```

Comment:
I suggest to create a couple of tests with random addresses generated from random mnemonic

Reply:
👍🏻

Reply:
[test that creates addresses from a mnemonic and validates the addresses](https://github.com/ayrat555/cryptopunk/pull/74/commits/a61d389a6da8c335fba28e1fae12f02365c26bcb)